selenium_tests/lti_emailer/page_objects/add_user_po.py

```diff
@@ -0,0 +1,43 @@
+__author__ = 'hic048'
```

Comment:
I'm not sure what page this is modeling -- is it intended to model the Canvas Users page? If so let's talk about a way to make that clearer and encapsulate the functionality better.